src/components/ExpensiTextInput/baseExpensiTextInputPropTypes.js

```diff
@@ -4,6 +4,9 @@ const propTypes = {
     /** Input label */
     label: PropTypes.string,
 
+    /** Name attribute for the input (only Web) */
```

Comment:
Since this is only used on web, should this just be added in index.js instead of including it here?

Reply:
Even though it is used for the Web, we will have to pass it to `ExpensiTextInput` irrespective of plaform. It has to be passed down to the BaseExpensiTextInput. we can simply pass null as the default value for native but still, it needs to be passed.
so prop definition is necessary. Just it does not affect non-web platforms.